config/config.go

```diff
@@ -0,0 +1,145 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/Layr-Labs/eigenda-proxy/common"
+	"github.com/Layr-Labs/eigenda-proxy/config/eigendaflags"
+	eigendaflags_v2 "github.com/Layr-Labs/eigenda-proxy/config/eigendaflags/v2"
+	"github.com/Layr-Labs/eigenda-proxy/metrics"
+	"github.com/Layr-Labs/eigenda-proxy/server"
+	"github.com/Layr-Labs/eigenda-proxy/store"
+	"github.com/Layr-Labs/eigenda-proxy/store/generated_key/memstore"
+	"github.com/Layr-Labs/eigenda-proxy/verify/v1"
+	"github.com/urfave/cli/v2"
+
+	"github.com/Layr-Labs/eigenda/api/clients"
+	clients_v2 "github.com/Layr-Labs/eigenda/api/clients/v2"
+)
+
+// AppConfig ... Highest order config. Stores
+// all relevant fields necessary for running both proxy
+// & metrics servers.
+type AppConfig struct {
+	EigenDAConfig ProxyConfig
+	MetricsCfg    metrics.Config
+}
```

Comment:
wait so we actually run 2 servers on 2 separate ports?
Think we should get rid of this, and just load the metrics handler on the same port that the normal REST routes are. Don't think we need the extra complexity.

Reply:
it's pretty common to run metrics on another port. imo it makes more sense to do this since you can control ingress per port which allows an operator to delineate networking policies.

Reply:
very true. makes sense